src/components/Onboarding/OnboardingPage2.tsx

```diff
@@ -86,42 +90,62 @@ const Page2: React.FC<Props> = ({ setPage, setHomeData, homeData }) => {
             </Text>
             <TextInput
               style={styles.formInput}
-              value={outdoor}
+              value={outDoorArea}
               onChangeText={setOutdoor}
               keyboardType="number-pad"
             />
             <Text style={styles.formComponent}>Number of Bedrooms</Text>
             <TextInput
               style={styles.formInput}
-              value={bedrooms}
+              value={bedroomCount}
               onChangeText={setBedrooms}
               keyboardType="number-pad"
             />
             <Text style={styles.formComponent}>Number of Bathrooms</Text>
             <TextInput
               style={styles.formInput}
-              value={bathrooms}
+              value={bathroomCount}
               onChangeText={setBathrooms}
               keyboardType="number-pad"
             />
-            <TouchableOpacity
-              style={{
-                backgroundColor: '#E96661',
-                marginTop: '25%',
-                width: 136,
-                height: 35,
-                borderRadius: 20,
-                alignItems: 'center',
-                justifyContent: 'center',
-              }}
-              onPress={nextPage}
-            >
-              <Text
-                style={{ color: 'white', fontSize: 16, fontWeight: 'bold' }}
+            {livableArea !== '0' && livableArea !== '' ? (
+              <TouchableOpacity
+                style={{
+                  backgroundColor: '#E96661',
+                  marginTop: '25%',
+                  width: 136,
+                  height: 35,
+                  borderRadius: 20,
+                  alignItems: 'center',
+                  justifyContent: 'center',
```

Comment:
Can we move this into the style sheet? If an element has its own unique color we can just do `style={{ backgroundColor: '#E96661', ...styles.opacity }}`